benchmark/comparisonTest/main.go

```diff
@@ -78,7 +85,8 @@ func main() {
 		CommitID:  commit,
 		Drivers:   drivers,
 	}
-	benchmarks.Run()
+	benchmarks.Run(ctx)
+	cancelCtx()
```

Comment:
could use defer, but chose not to, open to use it if people feel strongly

Reply:
I don't feel strongly but I'm curious why you chose not to

Reply:
I tried to come up with a good answer to this, and could not.  Therefore I moved them to defer - thank you!